apix_documentation_swagger_v3.yaml

```diff
@@ -147,6 +147,24 @@ info:
 
       `-H Authorization: APIAuth f0253d2a6a76ed12499e7878d7803ffb:nLet/JEZG9CRXHScwaQ/na4vsKQ=`
 
+    * ### Other headers
+
+      Aside from the Authorization header, you will also have to make sure that
+      your requests include the following headers:
+        * __Content-MD5__ It can be blank or a hash of digest base64.
+
+          `-H 'Content-MD5: 1B2M2Y8AsgTpgAmY7PhCfg=='`
+        * __Date__ It requires the same format used in the generation of the
+          checksum.
+
+          `-H 'Date: Tue, 24 Apr 2018 21:04:28 GMT'`
+        * __Content-Type__ Always should be application/json
+
+          `-H 'Content-Type: application/json'`
+        * __Accept__ Always should be application/vnd.regalii.v3.2+json
+
+          `-H 'Accept: application/vnd.regalii.v3.2+json'`
```

Comment:
I think the version (3.2) can be taken from a variable and use it to generate the docs. This way doesn't need to be hardcoded and when we change the version, this change automatically

Reply:
Is it ok with you if I create a ticket for that?

Reply:
Yes, we should create a ticket for everything regarding the documentation

Reply:
Jajaja yes, I meant if you agreed on creating a ticket vs figuring out how to do it right now

Reply:
https://arcusfi.atlassian.net/browse/USX-353